### PR TITLE
assign auto tag

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: cypress/integration/assign-automatic-tag.spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    })
+    // add a job probile
+    it('Add a job profile', function () {
+        cy.addJobprofile();
+    })
+
+    //visit inbox page and fetch latest emails
+    it('visit inbox page and fetch latest email', function () {
+        cy.server()
+        cy.route('GET',`http://localhost:8091/email/fetchByButton**`).as('fetch_emails')
```
Comment (reactexcel) [inline cypress/integration/assign-automatic-tag.spec.js]: why is url hard coded here?

File: cypress/integration/assign-automatic-tag.spec.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        cy.get("#fetchEmails").click();
+        cy.wait('@fetch_emails')
+    })
+
+    //check send email sucessfully fetched or not
+    it('check email is assigned to a tag or not', function () {
+        cy.get('#subnav .sub-title').contains(data.jobprofile);
+        cy.get('#subnav .subenav:first').find('a:first').should('not.have.text', 'all(0/0)');
+        cy.get('#subnav .subenav:first').find('a:first').click().then(function() {
+            cy.get('.emailstyle p').contains('ashutosh_m@excellencetechnologies.in')
```
Comment (reactexcel) [inline cypress/integration/assign-automatic-tag.spec.js]: no hard coding

Comment (reactexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: this test case is not clear. how are u checking the automatic tags are working here?

in this test case you are only check if emails are present or not that all??

File: cypress/integration/assign-automatic-tag.spec.js (hunk 1 above)
Comment (upexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: ok sir i will remove the hard coded part

File: cypress/integration/assign-automatic-tag.spec.js (hunk 2 above)
Comment (upexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: sir first i checked that added tag should exist in inbox then clicked on that tag and then checked if sent email should exist there.

Comment (upexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: in this test case i am checking, after clicking on tag email should exist in that tag.

Comment (reactexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: can you extend this test case to having 3 job profiles

Comment (upexcel) [reply cypress/integration/assign-automatic-tag.spec.js]: ok sir

**Pull request merged**